src/applications/vaos/appointment-list/components/AppointmentsPage/ExpressCareListItem.jsx

```diff
@@ -5,23 +5,30 @@ import { APPOINTMENT_STATUS } from '../../../utils/constants';
 
 export default function ExpressCareListItem({ appointment }) {
   const appointmentDate = moment.parseZone(appointment.start);
+  const canceled = appointment.status === APPOINTMENT_STATUS.cancelled;
 
   return (
     <li
       data-request-id={appointment.id}
       className="vaos-appts__card vads-u-display--flex vads-u-align-items--center"
     >
       <div className="vads-u-flex--1">
-        {appointment.status === APPOINTMENT_STATUS.cancelled && (
+        {canceled && (
           <span className="vaos-u-text-transform--uppercase vads-u-font-size--base vads-u-font-weight--bold vads-u-color--secondary-dark vads-u-margin-x--0 vads-u-margin-y--0">
             Cancelled
           </span>
         )}
         <h4 className="vads-u-font-size--h4 vads-u-margin-x--0 vads-u-margin-y--0">
           {appointmentDate.format('dddd, MMMM D')}
         </h4>{' '}
-        <span>A VA health care provider will follow up with you today.</span>
-        <br />
+        {!canceled && (
+          <>
+            <span>
+              A VA health care provider will follow up with you today.
+            </span>
+            <br />
+          </>
+        )}
```

Comment:
This instructional text only makes sense to show if the request hasn't been canceled.